dags/pegs-challenge-submission-dag.py

```diff
@@ -20,9 +20,9 @@
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
     "aws_conn_id": Param("AWS_TOWER_PROD_S3_CONN", type="string"),
     "revision": Param("0b45b1de981eb3e0c854bbbedbd6a570d95dbde2", type="string"),
-    "challenge_profile": Param("pegs_challenge_validate", type="string"),
+    "challenge_profile": Param("pegs_challenge_test", type="string"),
     "tower_conn_id": Param("PEGS_CHALLENGE_PROJECT_TOWER_CONN", type="string"),
-    "tower_view_id": Param("syn57373526", type="string"),
+    "tower_view_id": Param("syn52817032", type="string"),
```

Comment:
```suggestion
    "tower_view_id": Param("syn58942525", type="string"),
```

syn52817032 took me to the challenge website, but if we want the Submission View, then syn58942525 is the one to use.

Reply:
Thanks for double-checking!